### PR TITLE
Bump Heapster to 0.19.0 and Influxdb to 0.5 (0.9.4)

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

**Comment by mwielgus** [comment]
cc: @piosz

File: cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ spec:
         kubernetes.io/cluster-service: "true"
     spec: 
       containers: 
-        - image: gcr.io/google_containers/heapster_influxdb:v0.4
+        - image: gcr.io/google_containers/heapster_influxdb:v0.5
```
Comment (vishh) [inline cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml]: FYI: I added v0.6 to heapster which includes InfluxDB v0.9.6 which is supposed to consume much less disk space due to recent enhancements.

**Comment by vishh** [comment]
@mwielgus: Shall we use the latest InfluxDB instead? I suspect that the e2e tests will fail since they use the old InfluxDB APIs.

**Comment by vishh** [comment]
Ping @mwielgus

**Comment by mwielgus** [comment]
I created another release of Heapster - 0.19.0. We can use 0.9.6 (or 0.9 in general) but  we have to provide documentation on how to upgrade influxdb to the new version (we consulted this with @brendandburns) - storage engine and data layout has changed so it is not that straightforward.

**Comment by mwielgus** [comment]
I will update the pr shortly with 0.6 and test fix.

**Pull request closed without merging**